cmd/buildah/rmi.go

```diff
@@ -160,17 +164,24 @@ func deleteImages(ctx context.Context, systemContext *types.SystemContext, store
 				// because only a copy of the image state is returned
 				image, err = getImage(ctx, systemContext, image.ID, store)
 				if err != nil || image == nil {
-					return errors.Wrapf(err, "error getting image after untag %q", image.ID)
+					if lastError != nil {
+						fmt.Fprintln(os.Stderr, lastError)
+					}
+					lastError = errors.Wrapf(err, "error getting image after untag %q", image.ID)
 				}
 			}
 
 			isParent, err := imageIsParent(store, image.TopLayer)
 			if err != nil {
-				return err
+				if lastError != nil {
+					fmt.Fprintln(os.Stderr, lastError)
+				}
+				lastError = errors.Wrapf(err, "error judging image %q", image.ID)
```

Comment:
This error is kind of vague.  Perhaps "error determining if the image is a parent".  @umohnani8 WDYT?

Reply:
Agreed, "error determining if the image is a parent" is a more informative error.

Reply:
updated, PTAL.